app/frontend/src/components/Resources.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <v-container>
+    <h1 class="text-center my-8">Common Services Documentation</h1>
+    <v-card class="mx-auto mb-8" max-width="800">
+      <v-toolbar flat color="grey lighten-3">
+        <v-toolbar-title class="px-3">Resources</v-toolbar-title>
+      </v-toolbar>
+      <v-simple-table class="css-resp-table">
+        <template>
+          <tbody>
+            <tr>
+              <td>API Documentation</td>
+              <td>
+                <p>We recommend that you read the API documentation before running the Postman collection.</p>
+                <ul>
+                  <li class="my-2">
+                    Common Hosted Email Service (<a href="https://ches-master-9f0fbe-prod.pathfinder.gov.bc.ca/api/v1/docs">CHES</a>).
+                  </li>
+                  <li class="my-2">
+                    Common Document Generation Service (<a
+                      href="https://cdogs-master-idcqvl-prod.pathfinder.gov.bc.ca/api/v2/docs"
+                    >CDOGS</a>).
+                  </li>
+                </ul>
+              </td>
+            </tr>
+            <tr>
+              <td>POSTMAN Collection</td>
+              <td>
+                <p>Use the Common Service Postman Collection to verify your service client credentials and see the APIs in action.</p>
+                <ul>
+                  <li class="my-2"><router-link :to="{ name: 'RequestAccount' }">Get your service client</router-link>.
```

Comment:
I'd suggest only "service client" be as part of the hyperlink. Looks a bit strange to have an entire bullet point be a link when later points don't do the same thing.

Reply:
sure